package.json

```diff
@@ -10,7 +10,7 @@
     "test:watch": "jest --watchAll --coverage=false",
     "coveralls": "cat ./coverage/lcov.info | coveralls",
     "lint:staged": "lint-staged",
-    "prepublishOnly": "npm run build",
+    "prepublish": "npm run build",
```

Comment:
This is a temporary workaround to allow usage of the github repo as package.json dependency, revert before berging!